src/cli.rs

```diff
@@ -1,6 +1,9 @@
 use crate::settings::{read_policies_file, Policy};
 use anyhow::{anyhow, Result};
 use clap::{crate_authors, crate_description, crate_name, crate_version, App, Arg};
+use itertools::Itertools;
```

Comment:
Do we need itertools because we have to iterate inside of the macro below?

Reply:
No, itertools include a `sorted` function that we use so the output is stable: https://docs.rs/itertools/0.10.1/itertools/fn.sorted.html